apps/vmq_diversity/priv/vmq_bcrypt.schema

```diff
@@ -0,0 +1,30 @@
+%% @doc The pool_size specifies how many bcrypt operations that are
+%% allowed concurrently.
+{mapping, "vmq_bcrypt.pool_size", "bcrypt.pool_size", [
+                                                       {datatype, [integer, {enum, [logical_processors_online]}]},
+                                                       {default, 1},
+                                                       {include_default, 1}
+                                                      ]}.
+
+{translation,
+ "bcrypt.pool_size",
+ fun(Conf) ->
+         case cuttlefish:conf_get("vmq_bcrypt.pool_size", Conf) of
+             logical_processors_online ->
```

Comment:
should we call this 'auto'?

Reply:
yeah, let's pick sth which isn't mapped directly to the erlang term. I think `max` would be more descriptive, though.

Btw. I named the setting `vmq_bcrypt.pool_size`, it could also live in the global scope and be just something like `bcrypt_pool_size`. I'm not sure if we have some implicit rules about that?

Reply:
thinking a bit more, `max` isn't ideal either as you can of course have more ports ready than the number of logical cpus - perhaps `auto` isn't so bad after all. I'll change it to `auto`.